src/styles/utils/cards.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
```

Comment:
Do not hardcore the exact height of card, let elements inside dictate it. 
Also, it would be great to create a variable for width